src/api/subscriptions.js

```diff
@@ -286,9 +293,10 @@ export const ReplaceSkuProrationMode = {
 };
 
 /**
- * The product type. The Offers UI is rendered differently based on the
- * ProductType and 'product_type' parameter is passed to Payments flow to render
- * the GPay content accordingly. ProductType Defaults to 'SUBSCRIPTION'.
+ * The Offers/Contributions UI is rendered differently based on the
+ * ProductType. The ProductType parameter also passed to Payments flow to be
```

Comment:
The ProductType parameter is passed to the Payments flow, and then passed back to the Payments confirmation page

Reply:
Done